src/avram/migrator/runner.cr

```diff
@@ -70,9 +70,24 @@ class Avram::Migrator::Runner
       raise <<-ERROR
       #{message}
 
-        #{green_arrow} If you are on macOS  you can install postgres tools from #{macos_postgres_tools_link}
-        #{green_arrow} If you are on linux you can try running #{linux_postgres_installation_instructions}
-        #{green_arrow} If you are on CI or some servers, there may already be a database created so you don't need this command"
+      Try this...
+
+        ▸ #{green_arrow} If you are on macOS  you can install postgres tools from #{macos_postgres_tools_link}
+        ▸ #{green_arrow} If you are on linux you can try running #{linux_postgres_installation_instructions}
+        ▸ #{green_arrow} If you are on CI or some servers, there may already be a database created so you don't need this command"
+
+      ERROR
+    elsif (message = e.message) && messages.includes?("could not connect to database template1")
```

Comment:
I'm not sure what all the `template` mean, but my DB has `template0` through `template4`. Maybe we can just leave off the number just in case?

Reply:
Good idea!